external-crates/move/crates/move-compiler/src/parser/verification_attribute_filter.rs

```diff
@@ -15,23 +16,57 @@ use crate::{
 
 struct Context<'env> {
     env: &'env mut CompilationEnv,
+    is_source_def: bool,
+    current_package: Option<Symbol>,
 }
 
 impl<'env> Context<'env> {
-    fn new(compilation_env: &'env mut CompilationEnv) -> Self {
+    fn new(env: &'env mut CompilationEnv) -> Self {
         Self {
-            env: compilation_env,
+            env,
+            is_source_def: false,
+            current_package: None,
         }
     }
 }
 
 impl FilterContext for Context<'_> {
-    fn should_remove_by_attributes(
-        &mut self,
-        attrs: &[P::Attributes],
-        _is_source_def: bool,
-    ) -> bool {
-        should_remove_node(self.env, attrs)
+    fn set_current_package(&mut self, package: Option<Symbol>) {
+        self.current_package = package;
+    }
+
+    fn set_is_source_def(&mut self, is_source_def: bool) {
+        self.is_source_def = is_source_def;
+    }
+
+    fn should_remove_by_attributes(&mut self, attrs: &[P::Attributes]) -> bool {
```

Comment:
Add a similar comment explaining what this function does to that which accompanies function with the same name that decides on removing test-only module members?